### PR TITLE
Update logcli doc

### PR DESCRIPTION
**Pull request by daixiang0**
Reformat layout and add build way.

Signed-off-by: Xiang Dai <764524258@qq.com>

### CODE REVIEW

File: docs/logcli.md [hunk 1]
```diff
@@ -2,10 +2,31 @@
 
 Loki's main query interface is Grafana; however, a basic CLI is provided as a proof of concept.
 
-Once you have Loki running in a cluster, you can query logs from that cluster using the following commands:
+Once you have Loki running in a cluster, you can query logs from that cluster.
+
+## Installation
+
+### Get newest version
```
Comment (davkal) [inline docs/logcli.md]: Get latest version

File: docs/logcli.md [hunk 2]
```diff
@@ -2,10 +2,31 @@
...
 
 ```
 $ go get github.com/grafana/loki/cmd/logcli
+```
+
+### Build by yourself
```
Comment (davkal) [inline docs/logcli.md]: Build from source

File: docs/logcli.md [hunk 3]
```diff
@@ -29,6 +53,8 @@ Configuration values are considered in the following order (lowest to highest):
 
 The URLs of the requests are printed to help with integration work.
 
+### Detail
```
Comment (davkal) [inline docs/logcli.md]: Details

**Comment by daixiang0** [comment]
> Small language issues, otherwise good addition!

Done.

**Comment by daixiang0** [comment]
any update?

**Comment by daixiang0** [comment]
@davkal any update?

**Comment by daixiang0** [comment]
@davkal any update?

**Comment by daixiang0** [comment]
@davkal any update?

**Pull request merged**